### ISSUE TITLE
Java record formatting with annotation problem

### ISSUE THREAD

**Issue by VictorKwok** [apache/netbeans#4399]
### Apache NetBeans version

Apache NetBeans 14

### What happened

Whenever formating a record which using @JsonInclude, the formatting is not correct.

### How to reproduce

Please try to format this:
```
    @JsonInclude(JsonInclude.Include.NON_EMPTY)
    public record AccountBalance(
            String id,
            AccountType type,
            String name,
            String description,
            AssetType assetType,
            AccountUsage usage,
            Currency currency,
            BigDecimal balance,
            BigDecimal openBalance,
            OffsetDateTime openBalanceAt) {

    }
```

### Did this work correctly in an earlier version?

No

### Operating System

Windows 11

### JDK

Oracle JDK 18.0.1.1

### Apache NetBeans packaging

Apache NetBeans provided installer

### Anything else

It can be repeated everytime.

### Are you willing to submit a pull request?

No

### Code of Conduct

Yes

**Comment by neilcsmith-net** [comment]
Please share the output and what you expect the output to be. Thanks.

**Comment by VictorKwok** [comment]
For example, this is my record:
![image](https://user-images.githubusercontent.com/9064237/179879949-b6129d3f-47e9-4b9b-9ea3-86f0cb7745bc.png)
After right click -> Format
![image](https://user-images.githubusercontent.com/9064237/179879819-f415556d-2f55-4a31-9155-e98236068455.png)
Expected:
![image](https://user-images.githubusercontent.com/9064237/179879692-f013b51b-9f1c-449c-a59a-536dd71393f6.png)

**Comment by VictorKwok** [comment]
fixed by #5374
